### DISCUSSION TITLE
Sonar interface: "Hostiles at X'o'clock"

### DISCUSSION THREAD

**Discussion by James-E-A** [FakeFishGames/Barotrauma#8939] [Ideas and suggestions]
It seems a bit strange that you can't drag the Hostile report emblem onto the sonar.

This should be possible, and might even put the icon at the dropped location in-world for gunners to see, I would think.

**Comment by AshEdwardsDesign** [comment]
+1 this would be helpful. I'm often terrible at giving oclock directions when panicked 😄
